clusterfilehandlers/dfsbackends/oracle.php

```diff
@@ -1290,11 +1281,21 @@ protected function _query( $query, $fname = false, $reportError = true, $bindpar
                 if ( !oci_bind_by_name( $statement, $name, $bindparams[$name], -1 ) )
                 {
                     $this->error = oci_error( $statement );
-                    $this->_error( $query, $fname, $error );
```

Comment:
shuffled around a bit error handling to make it more clear. No functional changes in this